e2e/artefacts/k8s/nad.yaml

```diff
@@ -1,20 +1,12 @@
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
-  name: sriov-net1
+  name: nfnet
   annotations:
     k8s.v1.cni.cncf.io/resourceName: intel.com/ipu
 spec:
   config: '{
   "type": "nf-cni",
```

Comment:
nf-cni was the name we used in our own implementation of nf-cni. This needs to be the same as the one deployed on in the cluster.

Reply:
> nf-cni was the name we used in our own implementation of nf-cni. This needs to be the same as the one deployed on in the cluster.

This is just a place-holder for now for whatever cni binary is called for attaching the apf for NF. Once we know from DPU operator we can update it later